### DISCUSSION TITLE
Grupos locales liderando el proyecto

### DISCUSSION THREAD

**Discussion by nukeador** [PucelaBits/websegura#230] [General]
Hola a todos,

Llevo pensando tiempo que para poder mover este proyecto al siguiente nivel, necesitamos pensar cómo "empoderar" a grupos (formales o informales) locales interesados en #websegura a tomar la iniciativa y liderazgo en el proyecto. 

Moviendo ideas, recursos, presión y representación de forma local, que el proyecto sea movido por todos y no que dependa que de unos pocos que saquemos tiempo para ver cómo llevarlo al siguiente nivel.

- ¿Qué ideas se os ocurren? 
- ¿Participáis en algún grupo local que esté interesado?
- ¿Qué necesitaría este grupo para sentirse empoderado para mover el proyecto de forma local?

Un saludo.

**Comment by nukeador** [comment]
Una idea que se me ocurre es definir un contacto por cada provincia que haga de puente o facilitador.

Ahora mismo tenemos bastante gente que ha aportado a ciertas provincias y comunidades con lo que podríamos hablar para ver cómo apoyarles a movilizar más gente en su zona.

También creo que sería bueno tener una guía de acción local de #websegura,  donde se definan qué tipos de acciones se pueden hacer para potenciar el proyecto localmente y ejemplos de cómo hacerlo.

**Comment by viridicarium** [comment]
Yo estaría encantado de colaborar más activamente, pero mi barrera de entrada es mi conocimiento escaso-casi nulo del entorno github. De hecho, intenté añadir la provincia en la que resido y algunos enlaces, y no fui capaz. ¿alguna sugerencia de _manual / tutorial para dummies_?
Lo de la guía también es muy buena idea. Cuanto más clarito y masticado, mejor. Menos flecos sueltos!!
Y respecto a la pregunta que planteas _¿Qué necesitaría este grupo para sentirse empoderado para mover el proyecto de forma local?_ ... Pues habrá que seguir reflexionando
Gracias por el trabajo.
